pkgs/applications/video/olive-editor/default.nix

```
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, pkgconfig, which, qt5, frei0r, opencolorio, ffmpeg-full  }:
```

```suggestion
{ stdenv, fetchFromGitHub, pkgconfig, which, qmake, frei0r, opencolorio, ffmpeg-full  }:
```

All of `qt5` attributes are in scope so you can use them as an argument.
You should add the other ones aside from `qtbase`.